xcube_gen_vito/iproc.py

```diff
@@ -51,34 +48,3 @@ def get_extra_vars(self, dataset: xr.Dataset) -> Optional[Collection[str]]:
     def pre_process(self, dataset: xr.Dataset) -> xr.Dataset:
         # TODO (forman): clarify with VITO how to correctly mask the S2+ variables
         return super().pre_process(dataset)
-
-    def process(self,
-                dataset: xr.Dataset,
-                dst_size: Tuple[int, int],
-                dst_region: Tuple[float, float, float, float],
-                dst_resampling: str,
-                include_non_spatial_vars=False) -> xr.Dataset:
-        return reproject_crs_to_wgs84(dataset,
-                                      self.get_dataset_crs(dataset),
-                                      dst_size,
-                                      dst_region,
-                                      dst_resampling,
-                                      include_non_spatial_vars=include_non_spatial_vars)
-
-    @classmethod
-    def get_dataset_crs(cls, dataset: xr.Dataset) -> str:
-        proj_params = dataset["transverse_mercator"]
-
-        latitude_of_origin = proj_params.attrs["latitude_of_projection_origin"]
-        central_meridian = proj_params.attrs["longitude_of_central_meridian"]
-        scale_factor = proj_params.attrs["scale_factor_at_central_meridian"]
-        false_easting = proj_params.attrs["false_easting"]
-        false_northing = proj_params.attrs["false_northing"]
-
-        return get_projection_wkt("Some S2+ Tile",
-                                  "Transverse_Mercator",
-                                  latitude_of_origin,
-                                  central_meridian,
-                                  scale_factor,
-                                  false_easting,
-                                  false_northing)
```

Comment:
Now that you've removed `process()`, you should also remove the `get_extra_vars()` override. Also `pre_process()` override is not useful and should be removed.